src/Layout/redist/targets/GenerateLayout.targets

```diff
@@ -453,6 +456,14 @@
       ReplacementPatterns="$(ReplacementPattern)"
       ReplacementStrings="$(ReplacementString)" />
 
+    <!-- Work around https://github.com/dotnet/msbuild/issues/6215 by opting back into net7 behavior -->
+    <ReplaceFileContents
+      InputFiles="@(ToolRuntimeConfigPath)"
+      DestinationFiles="@(ToolRuntimeConfigPath)"
+      ReplacementPatterns="$(DisabledBinaryFormatterPattern)"
```

Comment:
I'm failing to find where this is being populated by the original value that we are replacing - is the key allways expected to be present?